src/utils/methods.js

```diff
@@ -0,0 +1,21 @@
+export function validateEmail(value) {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  }
+}
+
+export function validatePassword(value) {
+  if (!value) {
+    return 'Password is required';
+  }
+
+  if (value.length < 6) {
+    return 'At least 6 characters';
+  }
+}
```

Comment:
The function `validatePassword` does not return anything if the password is valid. Consider returning a success message or `null` to indicate that the password is valid.